test/rulesets/Base/base.jl

```diff
@@ -104,6 +104,13 @@
             test_frule(*, x, y)
             test_rrule(*, x, y)
         end
+        @testset "*($x, $y, ...)" for x in test_points, y in test_points
+            x, y = Base.promote(x, y)
```

Comment:
Why are we doing this?
these rules should work without promotion.

Reply:
I tried without this, and FiniteDifferences was not happy with mixed real + complex. The 2-arg case a few lines up has a comment to this effect.

Reply:
Can we open an issue there and link it back here?